uaa-server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM adoptopenjdk/openjdk11 as source
+RUN apt-get update && apt-get install -y git
+RUN git clone -b v75.0.0 https://github.com/cloudfoundry/uaa.git
+COPY uaa.yml /uaa/uaa/src/main/resources/uaa.yml
+WORKDIR /uaa
+RUN /uaa/gradlew
+
+FROM source as run
+WORKDIR /uaa
+EXPOSE 8080
+CMD ["/uaa/gradlew", "run"]
```

Comment:
Since you've set `WORKDIR`, you could, if you like, use relative commands, e.g `["./gradlew", "run"]`.  A benefit is that if you change working dir in future, you only have to change 1 line instead of 3.